engines/game.h

```diff
@@ -148,6 +148,17 @@ struct DetectedGame {
 	 */
 	GameSupportLevel gameSupportLevel;
 
+	/**
+	 * A list of extra keys to write to the configuration file
+	 */
+	Common::StringMap _extraConfigEntries;
+
+	/**
+	 * Convenience operator for adding entries to the _extraConfigEntries
+	 */
+	Common::String &operator[](const Common::String &key) {
```

Comment:
I'm not so sure it is a good idea to have this operator. This class defines what a game is when it was found by a detector, it does not behave like a bag of properties.

Reply:
Hmmm.. you're right, and we wouldn't want anyone to accidentally use it thinking it can be used to set other core/standard properties. I'll change it to an explicit method, and add in the comments as you suggested above visa ve. command line.